src/app/header/header.component.html

```diff
@@ -4,9 +4,14 @@
         <div class="topnav-right">
             <a routerLink="searchresults">Find Your Event</a>
             <a routerLink="/events">All Events</a>
-            <a routerLink="/addevent">Create Event</a>
+            <a *ngIf="loginService.isUserLoggedIn()" routerLink="/addevent">Create Event</a>
             <a routerLink="/adduser">Create Account</a>
         </div>
+        <div class ="topnav-left">
+          <a  *ngIf="!loginService.isUserLoggedIn()" routerLink="/login" class="nav-link" >Log-In</a>
+          <a  *ngIf="loginService.isUserLoggedIn()" routerLink="/logout" class="nav-link" >Logout</a>
+          <a  *ngIf="loginService.isUserLoggedIn()" routerLink="userprofile/:username" class="nav-link">Profile</a>
```

Comment:
This link does not work, it does not pull the logged in users info to the profile.